Detectors/TRD/base/include/TRDBase/PadPlane.h

```diff
@@ -13,8 +13,11 @@
 
 //Forwards to standard header with protection for GPU compilation
 #include "GPUCommonRtypes.h" // for ClassDef
-
+#include <array>
 #include "GPUCommonDef.h"
+#include <iostream>
```

Comment:
Probably a leftover from debugging? I think this line and the following two can be removed